src/js/stores/ChronosStore.js

```diff
@@ -90,19 +162,63 @@ class ChronosStore extends EventEmitter {
     }
   }
 
-  onVisibilityStoreChange() {
-    if (!VisibilityStore.isInactive() && this.shouldPoll()) {
-      startPolling();
+  getJobDetail(jobID) {
```

Comment:
`getJob` is more in line with [the pattern](https://github.com/dcos/dcos-ui/blob/master/src/js/stores/NodeHealthStore.js#L142) that we already have.